src/capi/typeobject.cpp

```diff
@@ -1022,14 +1023,13 @@ Box* slotTpGetattrHookInternal(Box* self, BoxedString* name, GetattrRewriteArgs*
                     } else
                         rtn->addGuard(0);
```

Comment:
It rewrites the case where we run into this case inside ```getattrInternalGeneric```: 
```c++
if (unlikely(rewrite_args && !descr && obj->cls != instancemethod_cls
         && rewrite_args->rewriter->aggressiveness() < 40
         && attr->interned_state == SSTATE_INTERNED_IMMORTAL)) {
	class Helper {
	public:
	    static Box* call(Box* obj, BoxedString* attr) { return obj->getattr(attr); }
	};

	RewriterVar* r_rtn = rewrite_args->rewriter->call(
	    false, (void*)Helper::call, rewrite_args->obj,
	    rewrite_args->rewriter->loadConst((intptr_t)attr, Location::forArg(1)));
	rewrite_args->setReturn(r_rtn, ReturnConvention::NOEXC_POSSIBLE);
	return Helper::call(obj, attr);
}
```
the call is marked as having no side effects to the additional guard don't abort the rewrite.